iree/turbine/kernel/wave/decompose_reduce_ops.py

```diff
@@ -66,9 +66,9 @@ def determine_shuffle_config(
         WORKGROUP_1,
         WORKGROUP_2,
     ] + induction_vars
-    offset = access_pattern.start.subs({k: 0 for k in ignore})
+    offset = safe_subs(access_pattern.start, {k: 0 for k in ignore})
```

Comment:
why safe_subs vs subs?

Reply:
I was getting errors `start` is `int` and not a sympy object during my experiments.